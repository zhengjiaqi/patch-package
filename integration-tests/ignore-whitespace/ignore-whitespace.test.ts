import { runIntegrationTest } from "../runIntegrationTest"

runIntegrationTest({
  projectName: "ignore-whitespace",
  shouldProduceSnapshots: true,
})
