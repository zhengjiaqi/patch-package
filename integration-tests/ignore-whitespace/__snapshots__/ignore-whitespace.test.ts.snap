// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ignore-whitespace: empty changeset when adding whitespace 1`] = `
"SNAPSHOT: empty changeset when adding whitespace
⁉️  Not creating patch file for package 'alphabet'
⁉️  There don't appear to be any changes.
END SNAPSHOT"
`;

exports[`Test ignore-whitespace: line a changed 1`] = `
"SNAPSHOT: line a changed
diff --git a/node_modules/alphabet/index.js b/node_modules/alphabet/index.js
index 7811d3b..454414b 100644
--- a/node_modules/alphabet/index.js
+++ b/node_modules/alphabet/index.js
@@ -1,4 +1,4 @@
-// a
+// patch-package
 // b   
 // c 
 // d
END SNAPSHOT"
`;
